### ISSUE TITLE
Timer does not start

### ISSUE THREAD

**Issue by paulmenzel** [hakimel/reveal.js#2236]
Creating a presentation with [Pandoc 2.3](https://pandoc.org/) starting the presentation windows by pressing the key *s*, the timer does not start. `git clone https://github.com/hakimel/reveal.js/` was used to copy the files to a directory.

The following is shown in the debugger console.

```
 [Mitteilungsdetails anzeigen/ausblenden] ReferenceError: Reveal is not defined[Weitere Informationen] notes.html:517:10
getTimings
file:///home/helmuth/src/reveal.js/plugin/notes/notes.html:517:10
setupTimer
file:///home/helmuth/src/reveal.js/plugin/notes/notes.html:574:20
handleConnectMessage
file:///home/helmuth/src/reveal.js/plugin/notes/notes.html:413:7
<anonym>
file:///home/helmuth/src/reveal.js/plugin/notes/notes.html:380:8
```

**Comment by paulmenzel** [comment]
Testing other example presentation from the reveal.js page, the timer works there.

I used the command below to create the presentation.

    pandoc -t revealjs -V revealjs-url=/home/helmuth/src/reveal.js --self-contained test.md -o test.html

**Comment by valencik** [comment]
I am also using revealjs via pandoc 2.3 and had success using the following patch: https://github.com/hakimel/reveal.js/pull/2104

**Comment by paulmenzel** [comment]
On 09/26/18 05:18, Andrew Valencik wrote:
> I am also using revealjs via pandoc 2.3 and had success using the
> following patch: https://github.com/hakimel/reveal.js/pull/2104

Using this branch of reveal.js, the timer indeed worked. Thank you!

**Comment by hakimel** [comment]
The fix in #2104 has been merged in now. It's currently only available in the reveal.js dev branch, but will be released in the next version.
